solution/ui/regulations/eregs-component-lib/src/components/JumpTo.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+    <div>
+        <div class="jump-to-label">Jump to Regulation Section</div>
+        <div class="form">
```

Comment:
We probably should try to keep this as a semantic `form` element if possible, since it is helpful for a11y and assistive tools like screen readers.

We're using forms in at least one other place in the app.  You can look at the example here:

https://github.com/CMSgov/cmcs-eregulations/blob/main/solution/ui/regulations/eregs-vite/src/components/SearchInput.vue

The main thrust is that instead of using `@click` to invoke the `getLink` method, you'd use `@submit.prevent` on the form itself.

Reply:
done